### ISSUE TITLE
[Documentation] adding missing accessibility details on SB

### ISSUE THREAD

**Issue by soykje** [adevinta/spark#810]
There had been an agreement about accessibility documentation: https://sparkui.vercel.app/?path=/docs/contributing-writing-packages-documentation--docs#6-accessibility But we missed it! So we should add these sections on each story :+1:
